hilt/migration-guide.md

```diff
@@ -463,6 +463,25 @@ When you have migrated all of the other dagger.android usages and are ready to
 remove this code, simply extend from `Application` and remove the overridden
 methods and the `DispatchingAndroidInjector` classes.
 
+You will also need to install `AndroidInjectionModule` and `AndroidSupportInjectionModule` to make dagger.android work.
+
+<div class="c-codeselector__button c-codeselector__button_java">Java</div>
+<div class="c-codeselector__button c-codeselector__button_kotlin">Kotlin</div>
+```java
+@InstallIn(ApplicationComponent.class)
+@Module(includes = {AndroidInjectionModule.class, AndroidSupportInjectionModule.class})
+public interface DaggerAndroidModule {
+}
+```
+{: .c-codeselector__code .c-codeselector__code_java }
+```kotlin
+@InstallIn(ApplicationComponent::class)
+@Module(includes = [AndroidInjectionModule::class, AndroidSupportInjectionModule::class])
```

Comment:
no need for AndroidSupportInjectionModule for quite some time now.
obsolete in https://github.com/google/dagger/commit/221f000e3d608637d5275b5c4650733c7668d450

Reply:
Fixed in https://github.com/google/dagger/pull/1924/commits/301b0f6ab7d234afdc57abede62ed0494faade13